Base/TimeInterval.h

```diff
@@ -0,0 +1,58 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file TimeInterval.h
+ *
+ * Created on July 17, 2019, 11:07 AM
+ *
+ */
+
+#pragma once
```

Comment:
please use the conventional `#ifndef ..` to make it consistent with other files. we haven't decided to move `#pragma once` in ogs5.

Reply:
Changed.